src/Administration/Resources/app/administration/src/app/component/context-menu/sw-context-button/index.js

```diff
@@ -94,7 +94,7 @@ Component.register('sw-context-button', {
         zIndex: {
             type: Number,
             required: false,
-            default: 9000,
+            default: 1100,
```

Comment:
then why not using the variable? :thinking:

Reply:
None of the z-index variables are set to an unnecessary high values, so setting the $z-index-context-menu to 9001 would cause it for example to overlap notifications, which I think is not a desired behaviour. 
Setting the z-index on this particular button would work, but every other button of this component without a specified z-index would still overlap its tooltips.

Reply:
ah telgis idea was to use the scss variable, but we are in JS and not in SCSS context here, so you can't easily access that varible

Reply:
haven't seen this :see_no_evil: sorry!